### PR TITLE
refactor: performance cw721 media info ( main )

### PR DESCRIPTION
**Pull request by phamphong9981**

### CODE REVIEW

File: src/services/cw721/cw721-media.service.ts [hunk 1]
```diff
@@ -282,16 +283,30 @@ export default class Cw721MediaService extends BullableService {
...
+            Bucket: BUCKET,
+            Key: fileName,
+          })
+          .promise();
+        return {
+          linkS3: S3_GATEWAY + fileName,
+          contentType: s3Object.ContentType,
+          key: fileName,
+        };
+      } catch {
```
Comment (peara) [inline src/services/cw721/cw721-media.service.ts:298]: catch cái gì

File: src/services/cw721/cw721-media.service.ts [hunk 2]
```diff
@@ -295,17 +296,22 @@ export default class Cw721MediaService extends BullableService {
...
+          );
+          let type: string | undefined = (
+            await FileType.fileTypeFromBuffer(mediaBuffer)
+          )?.mime;
+          if (type === 'application/xml') {
+            type = 'image/svg+xml';
+          }
+          return uploadAttachmentToS3(type, mediaBuffer);
+        } 
+          throw e;
```
Comment (peara) [inline src/services/cw721/cw721-media.service.ts:313]: lệch tab

File: src/services/cw721/cw721-media.service.ts [hunk 3]
```diff
@@ -295,17 +296,22 @@ export default class Cw721MediaService extends BullableService {
...
-        );
-        let type: string | undefined = (
-          await FileType.fileTypeFromBuffer(mediaBuffer)
-        )?.mime;
-        if (type === 'application/xml') {
-          type = 'image/svg+xml';
-        }
-        return uploadAttachmentToS3(type, mediaBuffer);
+      } catch (e) {
+        const error = e as AWSError;
```
Comment (peara) [inline src/services/cw721/cw721-media.service.ts:300]: cần check type

Comment (phamphong9981) [reply src/services/cw721/cw721-media.service.ts:300]: https://github.com/aws/aws-sdk-js/issues/2611

Comment (phamphong9981) [reply src/services/cw721/cw721-media.service.ts:300]: AWS Error em đọc nó ko check đc instanceof

**Pull request merged**